app/components/RoomList/index.js

```diff
@@ -339,6 +346,20 @@ class RoomList extends Component {
 									onClick={() => this.setState({ showRoomDeleter: true })}
 								/>
 							</Button.Group>
+							<span style={{ marginLeft: 16 }}>Status</span>
+							<Select
+								mode="multiple"
+								style={{ marginLeft: 16, width: '30%' }}
+								placeholder="Filter By Status..."
```

Comment:
I think just "Status" would be fine. it's pretty intuitive that these are filters and "Status" better follows the patterns of the other placeholders I think